client/dashboard/task-list/index.js

```diff
@@ -3,31 +3,108 @@
+			description: 'Install and manage your extensions directly from your Dashboard',
+			before: <Gridicon icon="sync" />,
+			after: <Gridicon icon="chevron-right" />,
+			onClick: noop,
+		},
+		{
+			title: __( 'Add your first product', 'woocommerce-admin' ),
+			description: 'Add products manually, import from a sheet or migrate from another platform',
+			before: <Gridicon icon="plus" />,
+			after: <Gridicon icon="chevron-right" />,
+			onClick: noop,
+		},
+		{
+			title: __( 'Personalize your store', 'woocommerce-admin' ),
+			description: 'Create a custom homepage and upload your logo',
+			before: <Gridicon icon="ink" />,
+			after: <Gridicon icon="chevron-right" />,
+			onClick: noop,
+		},
+		{
+			title: __( 'Set up shipping', 'woocommerce-admin' ),
+			description: 'Configure some basic shipping rates to get started',
+			before: <Gridicon icon="shipping" />,
+			after: <Gridicon icon="chevron-right" />,
+			onClick: noop,
+		},
+		{
+			title: __( 'Set up tax', 'woocommerce-admin' ),
+			description: 'Choose how to configure tax rates - manually or automatically',
+			before: <Gridicon icon="institution" />,
+			after: <Gridicon icon="chevron-right" />,
+			onClick: noop,
+		},
+		{
+			title: __( 'Set up payments', 'woocommerce-admin' ),
+			description: 'Select which payment providers you’d like to use and configure them',
+			before: <Gridicon icon="credit-card" />,
+			after: <Gridicon icon="chevron-right" />,
+			onClick: noop,
+		},
+	];
+};
+
+export default class TaskDashboard extends Component {
+	componentDidMount() {
+		document.body.classList.add( 'woocommerce-dashboard__body' );
+		document.body.classList.add( 'woocommerce-task-dashboard__body' );
+	}
+
+	componentWillUnmount() {
+		document.body.classList.remove( 'woocommerce-dashboard__body' );
+		document.body.classList.remove( 'woocommerce-task-dashboard__body' );
+	}
+
 	render() {
+		const siteUrl = wcSettings.siteUrl.replace( /(^\w+:|^)\/\//, '' );
+
 		return (
-			<div className="woocommerce-task-list">
-				<div className="woocommerce-task-list__header">
-					<H className="woocommerce-task-list__header-title">
-						{ __( 'Welcome to the WooCommerce Dashboard', 'woocommerce-admin' ) }
-					</H>
-					<H className="woocommerce-task-list__header-subtitle">
-						{ __(
-							"Here we'll guide you through the remaining tasks " +
-								'to get your store ready for launch',
+			<Fragment>
+				<div className="woocommerce-task-dashboard__sidebar" />
+				<div className="woocommerce-task-dashboard__header">
+					<div className="woocommerce-task-dashboard__header-site">{ siteUrl }</div>
+					<span className="woocommerce-task-dashboard__header-title">
+						{ __( 'Dashboard', 'woocommerce-admin' ) }
+					</span>
+					<div className="woocommerce-task-dashboard__header-connection">
+						<div className="woocommerce-task-dashboard__header-connection-avatar">
+							{ /* @todo This shoud display the user avatar if the store is connected. */ }
```

Comment:
missing an l in should

Reply:
Nice spot!  This shoud be fixed now :trollface: